10up/distributor
# Bump WP-Env version.

### Description of the Change

Bump wp-env to 1.5.0


Closes #

### How to test the Change

1. Run `npm i; npm run build`
2. If you've created an environment previously run `npm run env:destroy`
3. Run `npm env:start`
4. Confirm the test environment has been started by visiting http://localhost:8888/
5. Confirm the dev environment has started by visiting http://localhost
6. Login to each to ensure the Distributor plugin has been installled and activated
7. On the dev enviroment test sharing content between multiple sites.


### Changelog Entry

> Changed - Update WP-Env to 10.5.0


### Credits

Props @peterwilsoncc, @Sidsector9, @jeffpaul 


### Checklist:
